.envrc

```diff
@@ -24,14 +24,14 @@ export VITE_APP_S3_DOCUMENTS_BUCKET='local-uploads'
 export VITE_APP_S3_QA_BUCKET='local-qa'
 export DATABASE_URL='postgresql://postgres:shhhsecret@localhost:5432/postgres?schema=public&connection_limit=5' # pragma: allowlist secret
 export EMAILER_MODE='LOCAL'
-export LD_SDK_KEY='this-value-must-be-set-in-local'
+export LD_SDK_KEY='sdk-9f6315a3-ad22-4af8-9e21-b5caf2e12b98'
```

Comment:
@pearl-truss oops - this value shouldn't be sent in envrc, its always read for local envrc or set in CI. lets regenerate new keys also now that this has been published.

Reply:
whoops whoops! I reset the values in LD